pkg/service/controller/ingressip/controller.go

```diff
@@ -81,8 +82,8 @@ type serviceChange struct {
 // TODO this should accept a shared informer
 func NewIngressIPController(kc kclientset.Interface, ipNet *net.IPNet, resyncInterval time.Duration) *IngressIPController {
 	eventBroadcaster := record.NewBroadcaster()
-	eventBroadcaster.StartRecordingToSink(&kcoreclient.EventSinkImpl{Interface: kc.Core().Events("")})
-	recorder := eventBroadcaster.NewRecorder(kapi.EventSource{Component: "ingressip-controller"})
+	eventBroadcaster.StartRecordingToSink(&kv1core.EventSinkImpl{Interface: kv1core.New(kc.Core().RESTClient()).Events("")})
+	recorder := eventBroadcaster.NewRecorder(kapi.Scheme, clientv1.EventSource{Component: "ingressip-controller"})
```

Comment:
I'm assuming this is a similar issue to https://github.com/kubernetes/kubernetes/pull/41322... did you sweep for other places we're hitting that error?